bin/reports/compile_member_counts_report.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# Usage:
+# bundle exec ruby bin/reports/compile_member_counts_report.rb <COST_FREQ> <OUTPUT_DIR>
+
+$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), "../..", "lib"))
+
+require "report/member_counts_report"
+require "fileutils"
+
+def make_outf(output_dir)
+  if output_dir.nil?
+    $stdout
+  else
+    FileUtils.mkdir_p(output_dir)
+    ymd = Time.new.strftime("%F")
+    File.join(output_dir, "member_counts_#{ymd}.tsv")
+    File.open(fn, "w")
+  end
+end
+
+if __FILE__ == $PROGRAM_NAME
```

Comment:
I think this is the appropriate amount of code to have in the bin-file. I don't think it makes sense for the lib-file to know/care about where it writes itself to. Maybe we should take a larger perspective approach to this.

Reply:
Agreed the split in responsibilities makes sense. Mainly I'm thinking about whether we want to have essentially all the code from bin scripts in lib rather than in the bin script itself, and how we ensure as much of it is under test as possible, but maybe there's a bigger question there about a common CLI rather than 5809234902834 bin scripts. I'm OK punting on this for now.